testing/xdg-desktop-portal-kde/APKBUILD

```diff
@@ -1,14 +1,14 @@
 # Contributor: Rasmus Thomsen <oss@cogitri.dev>
-# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
+# Maintainer: Bart Ribbers <bribbers@disroot.org>
 pkgname=xdg-desktop-portal-kde
 pkgver=5.16.1
-pkgrel=0
+pkgrel=1
 pkgdesc="Qt/KF5 implementation of xdg-desktop-portal"
 url="https://phabricator.kde.org/source/xdg-desktop-portal-kde"
 arch="all"
 license="GPL-3.0-or-later"
-makedepends="cmake qt5-qtbase-dev glib-dev kio-dev kwayland-dev libepoxy-dev
-	pipewire-dev kcoreaddons xdg-desktop-portal extra-cmake-modules cups-dev"
+makedepends="extra-cmake-modules qt5-qtbase-dev glib-dev kio-dev kwayland-dev libepoxy-dev
+	pipewire-dev kcoreaddons xdg-desktop-portal cups-dev"
```

Comment:
I don't really see the reason for this change?

Also, maybe you could take a look at qt5-qtbase? I think that the cups-dev dependency actually belongs to it.

Reply:
`extra-cmake-modules` already depends on `cmake`, there isn't really a need to declare it separately. All other KDE packages already do it like this, this is just making it do the same.

Reply:
Oh, oops, didn't realise you replace `cmake` here, thought you just moved e-c-m for some reason